### PR TITLE
Relax version constraint for coq.

### PR DESCRIPTION
**Pull request by bschommer**
As said, ocaml 4.06 and coq work.

### CODE REVIEW

**Comment by RalfJung** [comment]
Maybe we should instead just revert to the old state? That was just `available: [ocaml-version >= "4.02.3" ]`.

**Comment by bschommer** [comment]
That is actually the better idea.

File: packages/coq/coq.8.7.0/opam [hunk 1]
```diff
@@ -56,4 +56,4 @@ depends: [
   "camlp5"
   "num"
 ]
-available: [ocaml-version >= "4.02.3" & ocaml-version < "4.06.0"]
+available: [ocaml-version >= "4.02.3" & ocaml-version]
```
Comment (RalfJung) [inline packages/coq/coq.8.7.0/opam]: This looks... funny? Is that even valid syntax? It's certainly not the way it used to look, with this additional `& ocaml-version`.

Comment (bschommer) [reply packages/coq/coq.8.7.0/opam]: No, I removed it.

**Comment by gasche** [comment]
Ok to merge as soon as the CI passes.

**Pull request merged**